static/audiofeed.yaml

```
@@ -1,3 +1,22 @@
+- title: '###-PILOT-### Zocken Ohne Zaster #1'
+  date: 2025-03-3T16:00
+  image: https://raw.githubusercontent.com/LucaNerlich/m10z/main/static/img/ZockenOhneZaster_Logo.jpg
+  description: |2-
+                  Liebe Freunde,
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)